rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/NodeSource.java

```diff
@@ -61,6 +61,10 @@
 
     private String eventType;
 
+    public NodeSource(String sourceName) {
+        this.sourceName = sourceName;
+    }
```

Comment:
Why not use lombok instead of adding setter and getter ?

Reply:
It is not setter but constructor